Specta/SpectaTests/TestHelper.m

```diff
@@ -1,15 +1,59 @@
 #import "TestHelper.h"
 
-XCTestSuiteRun *RunSpecClass(Class testClass) {
-  __block XCTestSuiteRun *result;
+@interface XCTestObservationCenter (Redeclaration)
```

Comment:
Can you explain about the observers? I'm not familiar with the concent.

Reply:
You don't want the test suite to report Spectas tests begin/end (and not XCTests), since Specta is using XCTest to test their `SPTSpec`s.